### PR TITLE
Add fulltext search

### PR DESCRIPTION
**Pull request by olegokunevych**

### CODE REVIEW

File: .vimrc [hunk 1]
```diff
@@ -31,6 +31,12 @@ Plug 'tpope/vim-surround'
 
 Plug 'w0rp/ale'
 
+" Fulltext search
+" Requirements:
+" brew install the_silver_searcher
+Plug '/usr/local/opt/fzf'
+Plug 'junegunn/fzf.vim'
```
Comment (AlexBeznoss) [inline .vimrc:38]: Add it to `README`, plz

Comment (olegokunevych) [reply .vimrc:38]: Updated, sorry for the delay

Comment (Miroshni4enko) [inline .vimrc:38]: We need update it to `Plug 'junegunn/fzf', { 'dir': '~/.fzf', 'do': './install --all' }`
or delete and add to requirements `brew install fzf`.
btw it's cool plugin, thanks @olegokunevych

Comment (mpmenne) [reply .vimrc:38]: @Miroshni4enko do you want to make that change?

**Review by mpmenne — APPROVED** [review]

File: .vimrc (hunk 1 above)
Comment (Miroshni4enko) [reply .vimrc:38]: My bad, everything is ok with this line.

**Pull request merged**